faker/tests/requirements.txt

```diff
@@ -0,0 +1 @@
+mock==1.0.1
```

Comment:
You're welcome! Could you cut a release with this change? We're waiting on it to merge some changes for our test suite.